SETUP/smoketests/pageload-smoketest.sh

```diff
@@ -0,0 +1,253 @@
+    'api/index.php?url=v1/projects/states'
+    'api/index.php?url=v1/projects/pagerounds'
+    'api/index.php?url=v1/projects/charsuites'
+    'api/index.php?url=v1/projects/specialdays'
+    'api/index.php?url=v1/projects/imagesources'
+    'api/index.php?url=v1/projects/holdstates'
+
+    'api/index.php?url=v1/queues'
+    'api/index.php?url=v1/queues/500'
+    'api/index.php?url=v1/queues/500/stats'
+    'api/index.php?url=v1/queues/500/projects'
+
+    'api/index.php?url=v1/stats/site'
+    'api/index.php?url=v1/stats/site/projects/stages'
+    'api/index.php?url=v1/stats/site/projects/states'
+    'api/index.php?url=v1/stats/site/rounds'
+    'api/index.php?url=v1/stats/site/rounds/P3'
+)
+
+FAQ_URLS=(
+    'faq/doc-copy.php'
+    'faq/feeds_sdk/index.php'
+    'faq/font_sample.php'
+    'faq/privacy.php'
+    'faq/prooffacehelp.php'
+    'faq/scanning.php'
+    'faq/simple_proof_rules.php'
+    'faq/site_progress_snapshot_legend.php'
+    'faq/translate.php'
+    'faq/wordcheck-faq.php'
+)
+
+MISC_URLS=(
+   # 'feeds/backend.php' # Disabled. It writes to a file
+   'locale/debug_ui_language.php'
+   'locale/translators/index.php'
+)
+
+QUIZ_URLS=(
+    'quiz/index.php'
+    'quiz/start.php'
+    'quiz/generic/hints.php?quiz_page_id=p_greek_4&error=G_n_u&number=1'
+    'quiz/generic/main.php?quiz_page_id=p_greek_1'
+    'quiz/generic/orig.php?quiz_page_id=p_greek_2'
+    'quiz/generic/proof.php?quiz_page_id=p_greek_3'
+    'quiz/generic/returnfeed.php?quiz_page_id=p_thorn'
+    'quiz/generic/right.php?quiz_page_id=p_fraktur'
+    'quiz/generic/wizard/default_messages.php'
+    'quiz/generic/wizard/start.php'
+    'quiz/generic/wizard/new_quiz.php'
+    # TODO: Needs SESSION data
+    #'quiz/generic/wizard/general.php'
+    #'quiz/generic/wizard/checks.php'
+    #'quiz/generic/wizard/messages.php'
+    #'quiz/generic/wizard/output.php'
+    #'quiz/generic/wizard/output_quiz.php'
+    #'quiz/generic/wizard/quiz_pages.php'
+
+    'quiz/tuts/tut_p_aeoe_1.php'
+    'quiz/tuts/tut_p_aeoe_2.php'
+    'quiz/tuts/tut_p_basic_1.php'
+    'quiz/tuts/tut_p_basic_2.php'
+    'quiz/tuts/tut_p_basic_3.php'
+    'quiz/tuts/tut_p_basic_4.php'
+    'quiz/tuts/tut_p_basic_5.php'
+    'quiz/tuts/tut_p_fraktur.php'
+    'quiz/tuts/tut_p_greek_1.php'
+    'quiz/tuts/tut_p_greek_2.php'
+    'quiz/tuts/tut_p_greek_3.php'
+    'quiz/tuts/tut_p_greek_4.php'
+    'quiz/tuts/tut_p_greek_5.php'
+    'quiz/tuts/tut_p_mod1_1.php'
+    'quiz/tuts/tut_p_mod1_2.php'
+    'quiz/tuts/tut_p_mod1_3.php'
+    'quiz/tuts/tut_p_mod1_4.php'
+    'quiz/tuts/tut_p_mod1_5.php'
+    'quiz/tuts/tut_p_mod2_1.php'
+    'quiz/tuts/tut_p_mod2_2.php'
+    'quiz/tuts/tut_p_mod2_3.php'
+    'quiz/tuts/tut_p_mod2_4.php'
+    'quiz/tuts/tut_p_mod2_5.php'
+    'quiz/tuts/tut_p_old_1.php'
+    'quiz/tuts/tut_p_old_2.php'
+    'quiz/tuts/tut_p_old_3.php'
+    'quiz/tuts/tut_p_thorn.php'
+)
+
+TEAMS_URLS=(
+    'stats/members/jointeam.php?tid=44' # NB performs an action
```

Comment:
Is there anyway we could get this list from a directory scan perhaps so we don't have to remember to add new pages to this list?

Reply:
We probably could, but I kinda don't want to, because I want us to exercise at least some of the 'real' code on each page, and not just error out at the top of the file complaining about invalid parameters.